modules/ROOT/pages/salesforce/salesforce-connector-97.adoc

```diff
@@ -0,0 +1,1274 @@
+
+=== SOAP API
+
+The Salesforce http://www.salesforce.com/us/developer/docs/api/index.htm[SOAP API] provides secure access to your organization's information on Salesforce. Most of the operations that the Salesforce connector performs map to this API.
+
+All the Salesforce operations performed through the SOAP API have an optional parameter called "Headers" that can take any of the following https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/soap_headers.htm[Salesforce SOAP Headers]:
+AllOrNoneHeader, AllowFieldTruncationHeader, AssignmentRuleHeader, CallOptions,
+EmailHeader, LocaleOptions, MruHeader, OwnerChangeOptions, QueryOptions,
+UserTerritoryDeleteHeader, DuplicateRuleHeader
+
+=== Apex REST API
+
+The Salesforce https://developer.salesforce.com/page/Creating_REST_APIs_using_Apex_REST[Apex REST API] creates your own REST-based web services using Apex. It has all of the advantages of the REST architecture, and provides the ability to define custom logic and includes automatic argument/object mapping.
+
+=== Bulk API
+
+The Salesforce https://www.salesforce.com/us/developer/docs/api_asynch/[Bulk API] quickly and securely load batches of your organization's data into Salesforce. See also: <<Load Data in Batches>>.
+
+=== Streaming API
+
+The Salesforce http://www.salesforce.com/us/developer/docs/api_streaming/[Streaming API] securely receives notifications for changes to your organization's information in Salesforce. See <<Receive Inbound Data From Salesforce>> for more information on the use of the Streaming API.
+
+=== Metadata API
+
+The Salesforce http://www.salesforce.com/us/developer/docs/api_meta/[Metadata API] manages customizations and build tools that work with the metadata model, not the data itself.
+
+== Configure in Design Center
+
+. In Anypoint Platform > *Design Center*, click *Create* > *Mule Application*.
+. In the first Mule Application card, if you want a Salesfore event to initiate access to your application (that is, to trigger your application), choose one of the Salesforce operations. Alternatively, you can use an *HTTP Listener* or *Scheduler* trigger.
++
+Salesforce operations:
++
+* *On Deleted Object* - Delete an object.
+* *On Modified Object* - Modify an object.
+* *On New Object* - Create a new object. Possible object types are Contact, Lead, Account, Contracts, Opportunity, and custom objects. The object types correspond to the Salesforce object you want to create, modify, or delete.
+* *Replay Channel* - Subscribe to a streaming channel. This trigger provides channel-related notification settings for new events and events stored by Salesforce within the 24-hour retention period. Events that take place in the specified channel trigger the flow.
++
+NOTE: A channel or topic must be published to Salesforce before a subscription to the channel can be created.

```

Comment:
Maybe "...before you can subscribe to the channel" or "before you can create a subscription to the channel"

Reply:
Fixed as requested